src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -388,18 +389,48 @@ private static void verifyConfig(Configuration testConfig,
     private void verifyViolations(Configuration config,
                                   String file,
                                   InputConfiguration inputConfiguration) throws Exception {
+        final List<String> actualViolations = getActualViolationsForFile(config, file);
+        final List<ParsedViolation> parsedViolations = inputConfiguration.getViolations();
+        verifyViolationLines(actualViolations, parsedViolations);
+        verifyViolationMessages(actualViolations, parsedViolations);
```

Comment:
lets compare lines and messages in one method simultaneously

Reply:
maybe we will have to convert `ParsedViolation` object to regex and match against actual violation

Reply:
done